src/main/java/no/ssb/guardian/maskinporten/config/MaskinportenClientConfig.java

```diff
@@ -1,31 +1,40 @@
-package no.ssb.maskinporten.guardian.config;
+package no.ssb.guardian.maskinporten.config;
 
 import io.micronaut.context.annotation.EachProperty;
 import io.micronaut.context.annotation.Parameter;
 import lombok.Data;
 
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
 import java.util.HashSet;
 import java.util.Set;
 
 @Data
 @EachProperty(value = "maskinporten.clients")
+@Valid
 public class MaskinportenClientConfig {
 
     /** clientId is the maskinporten client ID from samarbeidsportalen, typically a UUID */
+    @NotNull
     private String clientId;
 
     /** description is an optional string that describes some contextual information about the maskinporten client. Only used for documentation purposes. */
     private String description;
 
+    // TODO: Validate ending slash using bean validation
```

Comment:
@Pattern(
            regexp = ConfigValidator.AUDIENCE_VALIDATOR,
            message = "Invalid audience: " + ConfigValidator.INVALID_AUDIENCE_MESSAGE
    )

Reply:
See comment regarding this below